src/components/form-editor/form-editor.component.tsx

```diff
@@ -221,18 +234,64 @@ const FormEditorContent: React.FC<TranslationFnProps> = ({ t }) => {
       }
     }
   }, [stringifiedSchema, updateSchema, resetErrorMessage]);
-
+  const mergedSchemaForPreview = useMemo(() => {
+    if (!schema) return schema;
```

Comment:
Better to be explicit here.

```suggestion
    if (!schema) return null;
```